Translation of comments, readme, and code documentation to English  (not sphinx) (#1)
luk-f/pyCFOFiSAX: README.md

> # CFOF Calculation via iSAX

Reviewer: ```suggestion
# CFOF score computation via _i_SAX
```

[![Documentation Status](https://readthedocs.org/projects/pycfofisax/badge/?version=main)](https://pycfofisax.readthedocs.io/fr/main/?badge=main)

[![PyPI](https://github.com/luk-f/pyCFOFiSAX/actions/workflows/python-publish.yml/badge.svg)](https://github.com/luk-f/pyCFOFiSAX/actions/workflows/python-publish.yml)

> Project Thesis Lucas \

Reviewer: ```suggestion
Lucas' PhD project \
```

> Project Thesis Lucas \

Reviewer: ```suggestion
Lucas' PhD projects \
```

2017-2020

## To start

### Installation via PyPI

Run `pip install pyCFOFiSAX` or `python3 -m pip install pyCFOFiSAX`.

### Manual installation

Run `pip install -r requirements.txt` or `python3 -m pip install -r requirements.txt`.

> The package `tslearn` requires `Cython`, `numba` and `llvmlite` (`pip` will automatically download these packages).

Reviewer: ```suggestion
The `tslearn` package requires `Cython`, `numba` and `llvmlite` (`pip` will automatically download these packages).
```

> The package `tslearn` requires `Cython`, `numba` and `llvmlite` (`pip` will automatically download these packages).

Reviewer: ```suggestion
The `tslearn` package requires `Cython`, `numba` and `llvmlite` (`pip` will automatically download these packages).
```

> However for `Cython`, it is [necessary to have a C compiler (as described on this site)](https://cython.readthedocs.io/en/latest/src/quickstart/install.html),

Reviewer: ```suggestion
However for `Cython`, it is [necessary to have a C compiler (as described on this website)](https://cython.readthedocs.io/en/latest/src/quickstart/install.html),
```

and the [package `llvmlite`](https://llvmlite.readthedocs.io/en/latest/) for this project, tested only with Python 3.7 and 3.8
that we recommend.

> Otherwise, the use of `conda` simplifies the installation (as recommended on the site of[tslearn](https://tslearn.readthedocs.io/en/latest/index.html)), because does not require

Reviewer: ```suggestion
Otherwise, the use of `conda` simplifies the installation (as recommended on the site of [tslearn](https://tslearn.readthedocs.io/en/stable/)), because it does not require
```

Reviewer: ```suggestion
Otherwise, the use of `conda` simplifies the installation (as recommended on the site of[tslearn](https://tslearn.readthedocs.io/en/latest/index.html)), because it does not require
```

> the installation of compiler c and packages `Cython` and `llvmlite`.

Reviewer: ```suggestion
the installation of C compiler and `Cython` and `llvmlite` packages.
```

> ### Use

Reviewer: ```suggestion
### Documentation
```

The documentation is available here (in French) : [https://pycfofisax.readthedocs.io/fr/main/](https://pycfofisax.readthedocs.io/fr/main/)

## Thanks

> Development as part of the work of :

Reviewer: ```suggestion
Development of :
```

 - [CFOF: A Concentration Free Measure for Anomaly Detection, par Fabrizio Angiulli](https://arxiv.org/abs/1901.04992),
 - [iSAX: Indexing and Mining Terabyte Sized Time Series, par Jin Shieh et Eamonn Keogh](http://www.cs.ucr.edu/~eamonn/iSAX/iSAX.html),
 - [iSAX 2.0: Indexing and Mining One Billion Time Series, par Alessandro Camerra, Themis Palpanas, Jin Shieh et Eamonn Keogh](https://www.cs.ucr.edu/~eamonn/iSAX_2.0.pdf).
 - [Scoring Message Stream Anomalies in Railway Communication Systems, par Lucas Foulon, Serge Fenet, Christophe Rigotti et Denis Jouvin](https://hal.archives-ouvertes.fr/hal-02357924/)

> Using the code of:

Reviewer: ```suggestion
Use of the code of:
```
